src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -123,6 +123,9 @@
   <data name="ContentTypeHeaderRequired" xml:space="preserve">
     <value>The "content-type" header is required.</value>
   </data>
+  <data name="CustomAuditHeaderTooLarge" xml:space="preserve">
+    <value>The value of the header {0} is {1} and is too long so will not be added to the audit logs.  The maximum allowed size is {2}.  Please retry the request with a smaller value.</value>
```

Comment:
It looks like {1} is the size of the header value, so this text would read something like, "The value of the header X-FOO is 13789 and is too long."  That seems a little awkward/confusing.  Should it be, "The size of the header {0} is {1}"?